elf2table/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+//! INTERNAL; DO NOT USE. Please use the `defmt` crate to access the functionality implemented here.
```

Comment:
`defmt` doesn't reexport this crate's contents, so `elf2table` should probably be unstable instead of internal